docs/content/config/security/fail2ban.md

```diff
@@ -19,6 +19,8 @@ This following configuration files from `/tmp/docker-mailserver/` will be copied
 - `fail2ban-jail.cf` -> `/etc/fail2ban/jail.d/user-jail.local`
 - `fail2ban-fail2ban.cf` -> `/etc/fail2ban/fail2ban.local`
 
+You might want to make a volume bind for the `fail2ban` db to persist at `/var/lib/fail2ban/fail2ban.sqlite3` between container re-creations.
```

Comment:
This might be confusing, as it's DMS default, to make it this way.

Reply:
I am not sure to understand the action I should take, do you mean that somehow it already persists in a volume ?

Reply:
I was wrong.

Our [default config](https://github.com/docker-mailserver/docker-mailserver/blob/master/docker-compose.yml) makes mailstate persistant:

`      - ./data/mailstate:/var/mail-state`

However, if you set [ONE_DIR](https://docker-mailserver.github.io/docker-mailserver/edge/config/environment/#one_dir) to 1 (which I did), all states (fail2ban etc) are consolidate into this single directory (and preserved during restarts) --> `mail-state`. This looks like this:

```
# ll /var/mail-state
drwxrwxr-x  2 root         root     4.0K Feb  1  2021 etc-postsrsd
drwxr-x---  8 amavis       amavis   4.0K Aug 22 22:54 lib-amavis
drwxr-xr-x  2 clamav       dovecot  4.0K May 29 01:01 lib-clamav
drwxr-xr-x  2 root         root     4.0K Aug 22 22:54 lib-dovecot
drwxr-xr-x  2 root         root     4.0K May 14 23:21 lib-fail2ban
drwxr-xr-x  2 postfix      postfix  4.0K Mar  4  2020 lib-postfix
drwx------  2 postgrey     opendkim 4.0K Feb  1  2021 lib-postgrey
drwxr-xr-x  2 root         root     4.0K Aug 24  2020 lib-shared
drwxr-xr-x  5 debian-spamd postgrey 4.0K Mar  5  2020 lib-spamassassin
drwxr-xr-x 20 postfix      root     4.0K Mar  4  2020 spool-postfix

```

Maybe we should promote `ONE_DIR=1` as default.

Reply:
Good to know, I also use `ONE_DIR`
Will have to investigate